### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by isrumnon** [comment]
@yogkumgit can you please review?

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Example 5 returns information for user accounts that have been assigned a design
 Indicates the Identity of the user account to be retrieved.
 
 For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account:
+(Note: The mentioned changes are currently applicable only for Tenants in Public Cloud without a
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:110]: @isrumnon - seems like an incomplete sentence, also instead of Public Cloud - use Commercial Environments only like done below.

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -262,6 +263,10 @@ In the Teams PowerShell Module version 3.0.0 or later, the format of the Assigne
 
 - In the Teams PowerShell Module version 3.0.0 or later, the output format of Policies has now changed from String to JSON type UserPolicyDefinition.
 
+- Filtering for null polices: Admins need to query for null polices by including an empty value.
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:266]: Instead of null policies, it would be better to call out the scenario here

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -596,7 +602,8 @@ In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the
 ## INPUTS
 
 ## NOTES
-The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
+- The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online.
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:605]: "these sections" here is confusing

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -596,7 +602,8 @@ In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the
 ## INPUTS
 
 ## NOTES
-The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
+- The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
+- These changes are currently only rolled out in Commercial Environments and currently **not** applicable to Government Environments
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:606]: "currently" repeating - would suggest rewording

**Review by pbafna03 — CHANGES_REQUESTED** [review]
Added a few comments

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: done

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 2 above)
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: done

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 3 above)
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: done

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 4 above)
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: done

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -107,6 +107,8 @@ Example 5 returns information for user accounts that have been assigned a design
 Indicates the Identity of the user account to be retrieved.
 
 For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account:
+(Note: These changes are currently only rolled out in Commercial Environments and currently **not** applicable to Government Environments)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:110]:
```suggestion
For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are currently only rolled out in commercial environments and are currently **not** applicable to government environments):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -176,6 +178,8 @@ The Filter parameter uses the same filtering syntax as the Where-Object cmdlet.
 The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later:
 
 In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes:
+(Note: These changes are currently only rolled out in Commercial Environments and currently **not** applicable to Government Environments)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:181]:
```suggestion
In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes
(note that these changes are currently only rolled out in commercial environments and are currently **not** applicable to government environments):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -433,6 +441,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Notes
 
 The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later:
+(Note: These changes are currently only rolled out in Commercial Environments and currently **not** applicable to Government Environments)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:442]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (note that these changes are currently only rolled out in commercial environments and are currently **not** applicable to government environments):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 8]
```diff
@@ -596,7 +606,8 @@ In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the
 ## INPUTS
 
 ## NOTES
-The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
+- The changes mentioned under the "-Identity" and "-Filter" sections are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
+- These changes are currently only rolled out in Commercial Environments and **not** applicable to Government Environments
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:608]:
```suggestion
- These changes are currently only rolled out in commercial environments and are **not** applicable to government environments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after @JohanFreelancer9 suggestions.

**Comment by isrumnon** [comment]
@yogkumgit done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks
